src/sap/fhir/model/r4/lib/FHIRRequestor.js

```diff
@@ -26,20 +26,22 @@ sap.ui.define([
 	 * @param {sap.fhir.model.r4.FHIRModel} oModel The FHIRModel
 	 * @param {boolean} bCSRF If the FHIR service supports the csrf token
 	 * @param {string} sPrefer In which kind the FHIR service shall return the responses described here https://www.hl7.org/fhir/http.html#2.21.0.5.2
+	 * @param {array}  aDefaultQueryParams The default query parameters to be passed to for all GET requests if there is no specific resource level params. It should be of type key:value pairs in the array E.g. {'_total':'accurate'}
```

Comment:
see jsdoc comment in the FHIRModel.js

Reply:
Right now we are using the configuration only for GET ALL .. Have updated the JSdoc..